### ISSUE TITLE
Incorrect reference to org.pmw.runtime in export-package

### ISSUE THREAD

**Issue by smarquard** [tinylog-org/tinylog#65]
I'm running into an issue using TinyLog >= 1.1 in OSGI container (1.0.2 works fine), and it seems to be a packaging issue, for example:

```
org.osgi.framework.BundleException: Unresolved constraint in bundle cv.lecturesight.util [19]: Unable to resolve 19.0: missing requirement [19.0] osgi.wiring.package; (osgi.wiring.package=org.pmw.tinylog.runtime)
```

It looks like the export-package definition here may be wrong:

https://github.com/pmwmedia/tinylog/blob/v1.3/tinylog/pom.xml#L130

```
<Export-Package>org.pmw.tinylog, org.pmw.runtime,
  org.pmw.tinylog.labelers, org.pmw.tinylog.policies,
  org.pmw.tinylog.writers</Export-Package>
```

as it refers to `org.pmw.runtime` instead of `org.pmw.tinylog.runtime` ?

**Comment by smarquard** [comment]
It looks like the tinylog MANIFEST.MF also needs to include:

Import-Package: javax.naming

before this all works properly for me.

**Comment by pmwmedia** [comment]
Yes, you are right. It should be `org.pmw.tinylog.runtime` instead of `org.pmw.runtime`. I have fixed it in commit 3cfa38417d2858014acf06c4bb71508118ce776b.

Why should `Import-Package: javax.naming` be required? This package is part of the Java runtime environment.

**Comment by smarquard** [comment]
Thanks for the quick fix!

I got this error (having first edited MANIFEST.MF manually to change org.pmw.runtime to org.pmw.tinylog.runtime):

```
g! ERROR: lecturesight-framesource-videofile (25): [lecturesight.framesource.videofile] The activate method has thrown an exception
java.lang.NoClassDefFoundError: javax/naming/NamingException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at org.pmw.tinylog.PropertiesLoader.findImplementations(Unknown Source)
	at org.pmw.tinylog.PropertiesLoader.readWriter(Unknown Source)
	at org.pmw.tinylog.PropertiesLoader.readWriters(Unknown Source)
	at org.pmw.tinylog.PropertiesLoader.readProperties(Unknown Source)
	at org.pmw.tinylog.Configurator.init(Unknown Source)
	at org.pmw.tinylog.Logger.<clinit>(Unknown Source)
	at cv.lecturesight.framesource.videofile.VideoFileFrameSourceFactory.activate(VideoFileFrameSourceFactory.java:51)
```

so I followed the suggestions from this similar question:

https://stackoverflow.com/questions/15239943/getting-java-lang-noclassdeffounderror-javax-naming-namingexception-error-when

and that made everything work. I'm afraid my OSGI skills are not quite up to telling whether it's TinyLog that should have the import-package, or whether it's some other config issue in the project or other bundles that's causing the NoClassDefFoundError.

The project is here:

https://bitbucket.org/cilt/lecturesight/src/a8652a0ce18682860120a0851163293be08cc500/runtime/bundles/system/?at=t/LS-225-calibration

Tinylog 1.0.2 worked fine.

**Comment by pmwmedia** [comment]
I have fixed the reported issues and some others that I found while testing the bundles with felix.

@smarquard Could you verify that the bundles are working on your OSGi environment? Afterwards I will release the fixes as version 1.3.2.

Bundles:
[jcl-binding-1.3.2.zip](https://github.com/pmwmedia/tinylog/files/1568458/jcl-binding-1.3.2.zip)
[log4j-facade-1.3.2.zip](https://github.com/pmwmedia/tinylog/files/1568459/log4j-facade-1.3.2.zip)
[slf4j-binding-1.3.2.zip](https://github.com/pmwmedia/tinylog/files/1568461/slf4j-binding-1.3.2.zip)
[tinylog-1.3.2.zip](https://github.com/pmwmedia/tinylog/files/1568462/tinylog-1.3.2.zip)
[tinylog-jboss-1.3.2.zip](https://github.com/pmwmedia/tinylog/files/1568463/tinylog-jboss-1.3.2.zip)
[tinylog-jul-1.3.2.zip](https://github.com/pmwmedia/tinylog/files/1568464/tinylog-jul-1.3.2.zip)

**Comment by smarquard** [comment]
Thank you! I've tested tinylog-1.3.2.jar and it works fine.
